executable/executable_test.go

```diff
@@ -111,7 +120,10 @@ func TestHasExited(t *testing.T) {
 func TestStdin(t *testing.T) {
 	e := NewExecutable("grep")
 
-	e.Start("cat")
+	err := e.Start("cat")
+	if err != nil {
+		return
```

Comment:
@ryan-gang why are we returning here? This is a test so we'd want the full test to run?